state/containerDB.go

```diff
@@ -371,6 +373,18 @@ func (css *containerStateService) ContainerDBSize() int {
 	return len(css.idTable)
 }
 
+func (css *containerStateService) ContainerIsRegistered(id string) bool {
```

Comment:
As you may recall, we have been cautious to ensure that per-container i/o operations have no impact on Sysbox's registration/unregistration process/throughput (i.e., these thread groups are mostly decoupled by design). But notice that you're coupling these two functional blocks in this code, as you are stopping the global `containerStateService` every time a `passthrough` lookup() arrives...

Either way, I hope that we don't need this extra complexity -- will wait for your comments on my points above...

Reply:
If that's an issue, I could have the handler call the container object's `IsRegistrationCompleted()` method instead. 

In that case we could probably remove the `ContainerIsRegistered()` method from the containerStateService object.

Reply:
I removed this method from the containerStateService, but left it in the container object.